### DISCUSSION TITLE
Model files not importing correctly

### DISCUSSION THREAD

**Discussion by skybo053** [assimp/assimp#5725] [Q&A]
I've been attempting to import models like the following:

`const aiScene *scene = importer.ReadFile("Models/fbx/Dragon 2.5_fbx.fbx", aiProcess_Triangulate | aiProcess_FlipUVs);`

This always gives me a valid `scene` object but nothing is ever populated in the object.  Viewing in Visual Studio's debugger shows:

![image](https://github.com/user-attachments/assets/1258ae49-e677-4b2f-98ac-d2897ab2c8e7)

And opening `mRootNode` shows:

![image](https://github.com/user-attachments/assets/8640bbf3-0bfb-4dc2-8457-280bec438401)

I have tried using pre built binaries and building from source from several different tags. I've also tried several different model formats, but the result is always the same. Has anyone run into this issue?

**Comment by kimkulling** [comment]
Do you have any kind of log?

**Reply by skybo053** [reply]
I'm not getting any error logs.  I am checking the scene object with:
`if(!scene || scene->mFlags & AI_SCENE_FLAGS_INCOMPLETE || !scene->mRootNode)`, but none of these are true, and I've tried checking `Importer::GetErrorString`, but this is always empty.
